src/pyobo/api/utils.py

```diff
@@ -46,3 +50,40 @@ def get_version(prefix: str) -> Optional[str]:
         return data["version"]
 
     return None
+
+
+def get_version_pins():
+    """Retrieve the resource version pins."""
```

Comment:
full docstring, with explanation of 1) how to set this variable and also 2) why you would want to do this